src/module_wrap.cc

```diff
@@ -27,15 +27,13 @@ using v8::Maybe;
 using v8::MaybeLocal;
 using v8::Module;
 using v8::Object;
-using v8::Persistent;
 using v8::Promise;
 using v8::ScriptCompiler;
 using v8::ScriptOrigin;
 using v8::String;
 using v8::Value;
 
 static const char* EXTENSIONS[] = {".mjs", ".js", ".json", ".node"};
```

Comment:
Since you're here, can you make this `static const char *const`?  The pointed-to strings are const now but the array itself is not.

Reply:
done.